### DISCUSSION TITLE
What's the problem

### DISCUSSION THREAD

**Discussion by Sh1men9** [tw93/Pake#708] [Q&A] [answered]
What's the problem
![image](https://github.com/tw93/Pake/assets/91168875/d4d28efa-1cc9-4ac0-8bea-897af47d5f50)

**Comment by tw93** [comment, marked as answer]
Try not using Chinese, just an English word

**Reply by Sh1men9** [reply]
I'll give it a try

**Reply by Sh1men9** [reply]
Problem solved
